src/style.css

```diff
@@ -0,0 +1,47 @@
+* {
+  margin: 0 auto;
```

Comment:
reset all margins on corresponding tags selectors, not in *. Do not reset to 0 auto, just 0 would be enough, otherwise you will center all block elements by default